pkg/agent/datamodel/defaults-cloud-controller-manager.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
we don't need this for agentbaker as well... this is on control plane side

Reply:
PR to remove it is here: https://github.com/Azure/AgentBaker/pull/193